pnpm-workspace.yaml

```diff
@@ -23,3 +23,5 @@ catalog:
   'oxc-parser': '0.69.0'
   'oxc-transform': '0.69.0'
   'tsdown': '0.11.1'
+  'rolldown': '1.0.0-beta.8-commit.534fde3'
```

Comment:
This actually changes from referencing local rolldown to npm rolldown. Is this expected?

We expect to reference local rolldown to see any effect immediately.

Reply:
WIP, not finish yet.